src/_settings.scss

```diff
@@ -0,0 +1,763 @@
+ *
+    <body>
+        <modal>
+            <modal-close-button>
+        <modal-underlay>
+        <header>
+        <main>
+        <footer>
+    </body>
+ *
+ * You can then update the Sass map in your project's `_settings.scss`
+ * partial. And be sure to use nested maps to make the system aware of
+ * stacking contexts, as having a flat map will soon become a headache to
+ * maintain.
+ *
+ * Applying `z-index`s with this system:
+ *
+    .c-header {
+        z-index: z(header);
+    }
+ *
+ * And for nested stacking contexts (defined via a nested map):
+ *
+    .c-modal__close-button {
+        z-index: z(modal-elements, close-button);
+    }
+ *
+ * N.B. Scally uses a Sass map (and nested maps) to apply this Z-Index system
+ * therefore we use a Sass function to make it easier to apply the map keys
+ * and retrieve nested map keys, see:
+ * **Mixins/Functions -> Retrieve from maps**.
+ */
+
+$scally-g-zindexes: (
+    "modal-elements": (
+        "close-button": 1
+    ),
+    "modal": 5,
+    "modal-underlay": 4,
+    "header": 3,
+    "main": 2,
+    "footer": 1,
+    "underneath": -1
+) !default;
+
+
+
+
+/* 6. HiDPI units (Global)
+   ========================================================================= */
+
+/**
+ * HiDPI units, currently being used in:
+ * **Mixins/Functions -> HiDPI background images**.
+ */
+
+$scally-g-hidpi-dpr: 1.3 !default;
+
+$scally-g-hidpi-dpi: ($scally-g-hidpi-dpr * 96) * 1dpi !default;
+
+$scally-g-hidpi-dppx: 1.3 * 1dppx !default;
+
+
+
+
+/* 7. HTML textual `input`s (Global)
+   ========================================================================= */
+
+$scally-g-textual-inputs: 'input[type="text"],
+                           input[type="search"],
+                           input[type="tel"],
+                           input[type="url"],
+                           input[type="email"],
+                           input[type="password"],
+                           input[type="month"],
+                           input[type="week"],
+                           input[type="time"],
+                           input[type="date"],
+                           input[type="datetime"],
+                           input[type="datetime-local"],
+                           input[type="number"]' !default;
+
+
+
+
+/* 8. Base (Local)
+   ========================================================================= */
+
+/**
+ * Booleans.
+ */
+
+$scally-base-apply-osx-font-smoothing: false !default;
+
+$scally-base-apply-link-transition: true !default;
+
+$scally-base-apply-style-to-disabled-and-readonly-states: true !default;
+
+$scally-base-apply-bottom-margin-to-paragraphs: false !default;
+
+$scally-base-apply-print-styles: true !default;
+
+
+/**
+ * Root element.
+ */
+
+$scally-base-root-element-background-color: $scally-g-color-white !default;
+
+
+/**
+ * Links.
+ */
+
+$scally-base-link-text-decoration: underline !default;
+
+$scally-base-link-text-decoration-on-hover: none !default;
```

Comment:
Yeah, so I guess it's safe to set `text-decoration: none;` to the hover/focus pseudo classes if `$scally-base-link-text-decoration` value is `underline`? 😕 

The current Scally Base link is a mess so for v3.0.0 I wanted to strip it right back, something like:

```scss
a {
    color: $scally-g-color-link;
    text-decoration: $scally-base-link-text-decoration;

    @if $scally-base-apply-link-transition {
        transition: all $scally-base-link-transition-duration $scally-base-link-transition-timing-function;
    }

    &:hover,
    &:focus {
        color: $scally-g-color-link-hover;
        text-decoration: $scally-base-link-text-decoration-on-hover;
    }
}
```